CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/composite/ContractCompositePlugin.java

```diff
@@ -81,10 +81,10 @@ public void selectItem(String item, final Graph graph, GraphElementType elementT
             if (compositeNodeState != null && compositeNodeState.comprisesAComposite()) {
                 return Arrays.asList("Contract Composite");
             } else {
-                return null;
+                return Arrays.asList();
```

Comment:
@sol695510 is there any reason your using this instead of `Collections.emptyList()` because I noticed for one of the files you have use it. If no particular reason then would you be able to standardise on `Collections.emptyList()`?

Reply:
I'll make the change.